integrations/FacebookPixel/browser.js

```diff
@@ -292,35 +281,32 @@ class FacebookPixel {
         }
       }, legacyTo);
     } else if (event === "Product Added") {
-      if (!isDefinedAndNotNullAndNotEmpty(prodId)) {
-        // not adding index, as only one product is supposed to be present here
-        logger.error("Product ID is missing.");
+      const contentIds = [];
+      const contents = [];
+
+      if (prodId) {
+        contentIds.push(prodId);
+        contents.push({
+          id: prodId,
+          quantity,
+          item_price: price,
+        });
       }
+      const productInfo = {
+        content_ids: contentIds,
+        content_type: this.getContentType(rudderElement, ["product"]),
+
+        content_name: prodName || "",
+        content_category: category || "",
+        currency: currVal,
+        value: useValue ? this.formatRevenue(value) : this.formatRevenue(price),
+        contents,
```

Comment:
Here also we are sending empty contents array.

Reply:
yes. No error is thrown